src/main/java/org/isf/opd/manager/OpdBrowserManager.java

```diff
@@ -154,7 +154,7 @@ protected void validateOpd(Opd opd, boolean insert) throws OHDataValidationExcep
 	 * @return the list of Opds. It could be <code>null</code>.
 	 * @throws OHServiceException 
 	 */
-	public ArrayList<Opd> getOpd(String diseaseTypeCode,String diseaseCode, GregorianCalendar dateFrom,GregorianCalendar dateTo,int ageFrom, int ageTo,char sex,char newPatient) throws OHServiceException {
+	public ArrayList<Opd> getOpd(String diseaseTypeCode,String diseaseCode, Date dateFrom, Date dateTo,int ageFrom, int ageTo,char sex,char newPatient) throws OHServiceException {
```

Comment:
This is unclear, because dateFrom and dateTo in queries should always consider time, but in this case are actually compared with visitDate that is actually a Date and persisted as DATE in the DB.

The point is: many time we save correctly the DATEandTIME of an event, but then we must be able to query not caring about time. There are different solution out there, I'm also asking this [here](https://github.com/informatici/openhospital-core/pull/203#issuecomment-692889320).